documentation/homework/week1/todo-app/src/App.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import Item from './components/todo-item'
+import './App.css';
+
+class App extends Component {
+  render() {
+    return (
+      <div className="App">
+        <header className="App-header">
+          <img src={logo} className="App-logo" alt="logo" />
+          <h2>This is Todo-Item App</h2>
+        </header> {/*the Date should be -> M/D/Y */}
+          <Item description='Get out of bed' deadLineDate={'11/13/2017'} isDone={true} />
+          <Item description='Brush teeth' deadLineDate={'11/14/2017'} />
```

Comment:
Hard coding these components for week 1 is fine, but you should start to think about how to render these programmatically inside a loop for the future.

Reply:
I see your point but we don't have Any inserting method to know what kind of data that we will receive